style.css

```diff
@@ -0,0 +1,415 @@
+    background: linear-gradient(to right, #000000 461px, #FFFFFF 460px);
+  }
+
+  .up-section__collection {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+        -webkit-box-pack: center;
+        -ms-flex-pack: center;
+    justify-content: center;
+        -webkit-box-align: center;
+        -ms-flex-align: center;
+    align-items: center;
+    padding: 0 40px;
+    }
+
+  .up-section__images {
+    z-index: 10;
+    height: 562px;
+    margin-top: 77px;
+  }
+
+  .up-section__collection-image img {
+    width: 549px;
+    height: 562px;
+        -o-object-position: 50% 50%;
+    object-position: 50% 50%;
+  }
+
+  .up-section__collection-description {
+    width: 475px;
+    margin-left: 45px;
+  }
+
+  .up-section__title {
+    padding-left: 10px;
+    margin-top: 108px;
+    margin-bottom: 2px;
+    width: 475px;
+    height: 164px;
+    font-weight: bold;
+    line-height: 82px;
+    font-size: 80px;
+  }
+
+  .up-section__description,
+  .up-section__contacts {
+    display: inline-block;
+  }
+
+  .up-section__description {
+    padding-left: 9px;
+  }
+
+  .up-section__btn {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+        -webkit-box-pack: justify;
+        -ms-flex-pack: justify;
+    justify-content: space-between;
+    width: 490px;
+    padding-left: 5px;
+    margin-top: 23px;
+    text-decoration: none;
+    color: #000;
+  }
+
+  .up-section__btn.up-section__btn_offer {
+    position: relative;
+    width: 232px;
+    padding-top: 14px;
+  }
+
+  .up-section__btn_offer:hover {
+    transform: scale(1.1);
+  }
+
+  .up-section__btn.up-section__btn_offer::after {
+    position: absolute;
+    top: 17px;
+    right: -3px;
+    display: block;
+    content: '';
+    width: 44px;
+    height: 12px;
+    background-image: url(../img/vector-1.png);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  .up-section__scrooll {
+    bottom: 39px;
+    width: 27px;
+    height: 46px;
+  }
+
+  .up-section__scrooll:hover {
+    transform: scale(1.05);
+  }
+
+  .up-section__contacts {
+    margin-top: 69px;
+    margin-bottom: 21px;
+    margin-left: 12px;
+  }
+
+  .social {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+        -webkit-box-pack: justify;
+        -ms-flex-pack: justify;
+    justify-content: space-between;
+    width: 153px;
+    list-style: none;
+  }
+
+  .social__item:hover {
+  transform: scale(1.05);
+  }
+}
+
+@media (max-width: 992px) {
+  .header__head {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+        -webkit-box-pack: justify;
+        -ms-flex-pack: justify;
+    justify-content: space-between;
+    max-width: 461px;
+    background: #fff;
+  }
+
+  .header__logo {
+    margin-top: 14px;
+    margin-left: 26px;
+    color: #000;
+  }
+
+  .header__lang {
+    display: none;
+  }
+
+  .up-section__btn {
+    display: none;
+  }
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/locoalin/pullRequest?prid=5676737)